index.js

```diff
@@ -1 +1,42 @@
-export function astroTypedoc() {}
+import { writeFile } from 'node:fs/promises'
+import { Application } from 'typedoc'
+
+const pageTemplate = ({ title }) => `
+---
+---
+<html lang="en">
+  <head>
+    <title>${title}</title>
+  </head>
+  <body>
+    <h1>This page is about ${title}</h1>
+  </body>
+</html>
+`
+
+export async function generateApiDocs({
+  entryPoints,
+  pagesDirectory = 'src/pages/docs',
+  tsconfig
+}) {
+  let types = await getTypes({ entryPoints, tsconfig })
+
+  for (let type of types) {
+    await writeFile(
```

Comment:
Do we need to add `src/pages/docs` to `.gitignore` if we create files on the disk?

Reply:
They can be a sort of snapshots, but for now we could add it to `.gitignore`, yep.

Reply:
We may also need to add to `.gitignore` all built files (like `dist`) from our `test/webistes-*`

Reply:
`dist/`s are already in their local `.gitignore` from Astro starter